src/extension.ts

```diff
@@ -430,11 +430,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         return new LanguageClient('Phan Language Server', serverOptions, clientOptions);
     };
 
+    const languageClients = analyzedProjectDirectories.map(createClient);
+
+    async function restartLanguageClients(): Promise<void> {
+        for (const client of languageClients) {
+            await client.stop();
+            context.subscriptions.push(client.start());
+        }
+    }
+    context.subscriptions.push(vscode.commands.registerCommand('phan.restartLanguageServer', restartLanguageClients));
```

Comment:
This seems right for adding a Restart command and the package.json seems like it's the right way to add a button